www/js/app.js

```diff
@@ -382,13 +382,16 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
         document.getElementById('formArticleSearch').style.display = '';
         document.getElementById('welcomeText').style.display = '';
         // Give the focus to the search field, and clean up the page contents
-        $("#prefix").val("");
-        $('#prefix').focus();
-        $("#articleList").empty();
-        $('#articleListHeaderMessage').empty();
+        document.getElementById('prefix').value = '';
+        document.getElementById('prefix').focus();
+        let articleList = document.getElementById('articleList');
+        let articleListHeaderMessage =  document.getElementById('articleListHeaderMessage');
+        while (articleList.firstChild) articleList.removeChild(articleList.firstChild);
+        while (articleListHeaderMessage.firstChild) articleListHeaderMessage.removeChild(articleListHeaderMessage.firstChild);
         document.getElementById('searchingArticles').style.display = 'none';
         document.getElementById('articleContent').style.display = 'none';
-        $("#articleContent").contents().empty();
+        let articleContent = document.getElementById('articleContent');
+        while (articleContent.firstChild) articleContent.removeChild(articleContent.firstChild);
```

Comment:
```suggestion
        let articleContent = document.getElementById('articleContent');
        let articleContentDoc = articleContent ? articleContent.contentDocument : null;
        while (articleContentDoc.firstChild) articleContentDoc.removeChild(articleContentDoc.firstChild);
```
Here is my suggestion. It needs testing.

Reply:
Thank you! I will test it.

Reply:
@dheerajdlalwani I also suggest you add a comment just above this block, so that devs know what this code is doing:

`// Empty and purge the article contents`